src/app/common/components/ErrorModal.tsx

```diff
@@ -12,7 +12,7 @@ import { ErrorCircleOIcon } from '@patternfly/react-icons';
 import { useHistory } from 'react-router-dom';
 import { AlertActions } from '../duck/actions';
 import { connect } from 'react-redux';
-import { PollingContext } from '../../home/duck/context';
+import { PollingContext } from '../../home/duck';
```

Comment:
This is throwing an error for me. Any reason we are changing the path here?

Reply:
The reason is because before we're defining a barrel file (index file loading deps) but we're not using it. If we use `.../home/duck/context` then there is no need for the index file.